### ISSUE TITLE
improve log output of UI tests

### ISSUE THREAD

**Issue by individual-it** [owncloud/client#8799]
the current output of the test logs in CI is hard to parse for humans. It's difficult to find out which scenarios failed and how. Basically the developer has to look for `ERROR` (capital case) to find the failing line.

- [ ] find a possibility to output the failed scenarios at the end of the test-run
- [ ] find a way of having an other  keyword than `ERROR`

@TheOneRing @jnweiger @swoichha @Talank other ideas how to improve test output readability?

**Comment by individual-it** [comment]
the cli option `--reportgen` might be useful https://doc.froglogic.com/squish/latest/rg-cmdline.html#rg-squishrunner-report-generators

**Comment by Talank** [comment]
if we run the tests with `--reportgen json1.2,<report-dir>`, then we get the test results in json format where the failed tests contains the key and value `result: 'FAIL'`. May be we can parse the json data and get the name of failing scenario only.

![Screenshot from 2021-08-04 14-28-05](https://user-images.githubusercontent.com/27364813/128151374-7aa16232-d3a4-49f0-8291-9726b9e7f906.png)

**Comment by Talank** [comment]
The log on failure is now better and more insightful. Each failure is shown in a JSON object with following structure.
```json
{
  "Error Details": "Actual error message, cause of the error",
  "Feature": "NAME OF FEATURE IN WHICH THE FAILING TEST IS",
  "Feature File": "NAME OF THE FILE IN WHICH THE FAILING TEST IS",
  "Scenario": "FAILING TEST SCENARIO",
  "Test Step": "STEP AT WHICH THE ERROR PERSIST"
}
```
So, closing I'm this issue.
